src/locale/lang/en_US.js

```diff
@@ -1,9 +1,16 @@
 const en_US = {
-  lang:'简体中文',
+  lang: '简体中文',
   home: 'Home',
   dashboard: 'Dashboard',
-  signIn:'Sign in',
-  signUp:'Sign up',
-  signOut:'Sign out'
+  signIn: 'Sign in',
+  signUp: 'Sign up',
+  signOut: 'Sign out',
+  defaultEmail: 'Default Email',
+  notification: 'Notifications',
+  setting: 'Setting',
+  successAdd: 'Successfully added!',
```

Comment:
Considering add prefix "notification" or "notificationServerchan" to related items. The current naming is a bit confusing (inputName for websentry is about ServerChan name).